velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -35,6 +35,23 @@ class ReferenceQueryRunner {
       const std::string& sql,
       const std::vector<RowVectorPtr>& input,
       const RowTypePtr& resultType) = 0;
+
+  /// Returns true if the underlying Query Runner supports
+  /// returning the results of a query in Velox Vector format. If supported
+  /// then 'executeVector' can be called to get results as Velox Vector.
+  virtual bool supportsVeloxVectors() const {
+    return false;
+  }
+
+  /// Executes SQL query returned by the 'toSql' method using 'input' data
+  /// and returns the results in RowVector format. Check if this function is
+  /// supported by calling the 'supportsVeloxVectors' function before calling.
+  virtual std::vector<RowVectorPtr> executeVector(
+      const std::string& sql,
+      const std::vector<RowVectorPtr>& input,
+      const RowTypePtr& resultType) {
+    VELOX_NYI("Not supported in this Reference Query Runner.");
```

Comment:
NYI means Not Yet Implemented. This is not the right macro to use here. Use VELOX_UNSUPPORTED instead. Then, the error message becomes redundant and can be removed.

Reply:
Makes sense.